code/model/ForumThread.php

```diff
@@ -302,7 +302,7 @@ static function notify(Post $post) {
 					$email = new Email();
 					$email->setFrom(Email::getAdminEmail());
 					$email->setTo($member->Email);
-					$email->setSubject('New reply for ' . $post->Title);
+					$email->setSubject(_t('Post.NEWREPLY', 'New reply for') . " " . $post->Title);
```

Comment:
I'll pull in this PR locally and cherry pick + update to push 0.8.0 release forward :+1: